### PR TITLE
Fixed.....

### PR DESCRIPTION
**Pull request by rhtv43**
@lexis-t :Please review the same...

### CODE REVIEW

**Comment by rhtv43** [comment]
@lexis-t : Hi Alexey,

Please review the changes so that I can merge the same into master branch.

File: lib/commonAPI/mediacapture/ext/platform/android/src/com/rho/camera/CameraObject.java [hunk 1]
```diff
@@ -139,8 +147,8 @@ public void onPictureTaken(byte[] data, Camera camera) {
             }
             bitmap.recycle();
             mPreviewActivity.finish();
-            
-        } 
+            mcameraActivity.finish();
```
Comment (lexis-t) [inline lib/commonAPI/mediacapture/ext/platform/android/src/com/rho/camera/CameraObject.java]: This is unneeded. mcameraActivity is same object as mPreviewActivity. You call finish() twice.

File: lib/commonAPI/mediacapture/ext/platform/android/src/com/rho/camera/CameraObject.java [hunk 2]
```diff
@@ -206,20 +214,18 @@ public void stopPreview() {
...
         	if(desired_width > 0 && desired_width <= maxSize.width && desired_height > 0 && desired_height <= maxSize.height){        	
         	 Camera.Size previewSize = getOptimalPreviewSize(parameters.getSupportedPictureSizes(), desired_width, desired_height);
         	 Logger.T(TAG, "Selected size: " + previewSize.width + "x" + previewSize.height);             
              parameters.setPreviewSize(previewSize.width, previewSize.height);
         	}
-        	else{
-        		//Handle error message
-        		intent.putExtra("error", "Please enter a valid desired value");        		
+        	else{ 
+        		parameters.setPreviewSize(320, 240);
```
Comment (lexis-t) [inline lib/commonAPI/mediacapture/ext/platform/android/src/com/rho/camera/CameraObject.java]: Why you decided to not inform developer about wrong parameter value? The behavior at hi-res devices will be strange?...

File: lib/commonAPI/mediacapture/ext/platform/android/src/com/rho/camera/CameraActivity.java [hunk 3]
```diff
@@ -107,4 +108,21 @@ public void onClick(View view) {
...
+        int volume = meng.getStreamVolume( AudioManager.STREAM_NOTIFICATION);
+
+        if (volume != 0)
+        {
+        	
+			if (_shootMP == null)
+            //    _shootMP = MediaPlayer.create(getBaseContext(), Uri.parse("file:///sdcard/malaya/sleep_away.3gpp"));
+				_shootMP = MediaPlayer.create(getBaseContext(), Uri.parse(musicPath));
+            if (_shootMP != null)
+                _shootMP.start();
```
Comment (lexis-t) [inline lib/commonAPI/mediacapture/ext/platform/android/src/com/rho/camera/CameraActivity.java]: You need to release media player object when stop playing.

**Comment by rhtv43** [comment]
@lexis-t : I have done the changes as suggested. Please review the same.

**Comment by rhtv43** [comment]
@lexis-t : Since the changes has to be reflected in the current build, I am merging the same.

**Pull request merged**